### ISSUE TITLE
[: 31: -eq: unexpected operator

### ISSUE THREAD

**Issue by soncco** [nicokaiser/node-monit#2]
When I tried to start node-service, I've got this

```
[: 31: -eq: unexpected operator
```

The line 31 has:

```
[ $UID -eq "0" ] && LOCAL_VAR_RUN=/var/run
```

I'm using Ubuntu 12.10

**Comment by nicokaiser** [comment]
Can you try replacing `#! /bin/sh` with `#! /bin/bash`?

On Ubuntu, you don't need SysVinit – you can use the Upstart scripts, as Ubuntu comes with Upstart.

**Comment by jdubie** [comment]
using `#! /bin/bash` worked for me

**Comment by soncco** [comment]
Thank you, works fine

**Comment by m-a-r-c-e-l-i-n-o** [comment]
I think it's worth noting that the issue has to do with "sh" defaulting to "dash" as of Ubuntu 6.10.

**Comment by nicokaiser** [comment]
see #1, this should be fixed now (058b546).
